blaze/server/serialization.py

```diff
@@ -227,9 +241,24 @@ def fastmsgpack_materialize(data, dshape, odo_kwargs):
 )
 
 
-all_formats = frozenset(
-    g for _, g in globals().items() if isinstance(g, SerializationFormat)
+fastmsgpack_trusted = SerializationFormat(
+    'fastmsgpack_trusted',
+    loads=partial(fastmsgpack_loads, object_hook=object_hook_trusted),
+    dumps=partial(fastmsgpack_dumps, default=json_dumps_trusted),
+    data_loads=fastmsgpack_data_loads,
+    data_dumps=fastmsgpack_data_dumps,
+    materialize=fastmsgpack_materialize,
 )
 
 
+most_formats = frozenset(g for _, g in globals().items()
```

Comment:
can we give this a more descriptive name? what would the user want to use this set for?